api/schema/aws_account_resource.go

```diff
@@ -32,7 +32,7 @@ var (
 						Mode: ImmutableAttributeMode,
 					},
 				},
-				"mode": StringResourceAttributeWithMode{
+				"access_mode": StringResourceAttributeWithMode{
```

Comment:
1. Update the example(s) whenever you change a schema. https://github.com/illumio/terraform-provider-illumio-cloudsecure/blob/0bc6af692610b7e69fcf4dcfb8831e13dca9aa9b/examples/resources/illumio-cloudsecure_aws_account/resource.tf#L9
1. Re-generate the doc afterwards.

Reply:
to better match the db field name, and i felt the documentation says it anyways that its access_mode.

Reply:
Did you mean to answer to [the other comment](https://github.com/illumio/terraform-provider-illumio-cloudsecure/pull/81#discussion_r1775774952)?